tests/neg/i8032.check

```diff
@@ -1,9 +0,0 @@
--- [E007] Type Mismatch Error: tests/neg/i8032.scala:1:15 --------------------------------------------------------------
-1 |val x: 1 | 2 = 3 // error
-  |               ^
-  |               Found:    (3 : Int)
-  |               Required: (1 : Int) | (2 : Int)
--- [E008] Not Found Error: tests/neg/i8032.scala:3:12 ------------------------------------------------------------------
-3 |val y = ???.map // error
-  |        ^^^^^^^
-  |        value map is not a member of Nothing
```

Comment:
No it was removed since the error message now contained an additional clause
```
Note that implicit extension methods cannot be applied because they are ambiguous;
  |both method charArrayOps in object Predef and method doubleArrayOps in object Predef provide an extension method `map` on (??? : => Nothing)
2 
```
but that clause was not deterministic (different conversion candidates showed up). I have now changed the reporting
so that the eplanation is dropped for arguments of type Nothing and brought the check file back.